local-api/formlabs-api-local-openapi.yaml

```diff
@@ -279,6 +279,15 @@ components:
         - DETECTED
         - MILLIMETERS
         - INCHES
+    LockModel:
+      type: string
+      description: The orientation constraints of the model, which auto-pack operations will use
```

Comment:
There are no other scene mutation APIs, right? I can't snap models to the BP, can I?

Either way, I'd consider expanding this beyond auto-pack. In PreForm model lockings prevent all forms of model pose manipulation.

Reply:
There are definitely other [scene mutation](https://formlabs-dashboard-api-resources.s3.amazonaws.com/formlabs-local-api-latest.html#tag/Modifying-a-Scene) endpoints. So specifically, a POST to /scene/models/<id> will change the model, and can specify a complete position and orientation.

Our thinking with this was that server-side locking should only affect the /auto/ stuff. It seems needless to block a client from directly setting a position when they could just... skip that. While a client who wants the opposite would need an annoying unlock/move/lock combo with delay and potential syncing issues.

So okay, at the very least behavior should be documented! And if you feel strongly it's the wrong behavior I'd be open to changing it.